### PR TITLE
Update Air India

### LINKED ISSUE #9311: Update Air India
### Brand Name

Air India

### Official Resources for Icon and Color

svg: [https://upload.wikimedia.org/wikipedia/en/e/e3/Air_India_Logo.svg](https://upload.wikimedia.org/wikipedia/en/e/e3/Air_India_Logo.svg)
color: #DD2842

### Additional Comments

Air India will use this logo starting December 2023.

### Contributing

Yes

### PR DESCRIPTION
**Pull request by wwwescape**
airindia

**Issue:** Closes #9311
**Similarweb rank:** [21,201](https://www.similarweb.com/website/airindia.com/#overview)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
 - I used the SVG from Wikipedia
 - I used Inkscape to scale and adjust
 - I used https://jakearchibald.github.io/svgomg/ to optimize

### CODE REVIEW

File: _data/simple-icons.json
Comment (adamrusted) [inline _data/simple-icons.json:376]: If sourced from Wikipedia, I'd update this entry to that for now.
```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:Air_India_2023.svg"
```

Looks like other than the specific 'rebrand' page - they've not yet updated their main website.

Comment (wwwescape) [reply _data/simple-icons.json:376]: Shall we keep this PR and any changes on hold till December 2023?

Comment (adamrusted) [reply _data/simple-icons.json:376]: Can do - is that when the rebrand is supposed to take effect?

**Comment by PeterShaggyNoble** [comment]
@wwwescape, Air India have now rolled the new branding out to their website, complete with SVGs. I _think_ we should be able to get away with using just, what they call, the "Vista" symbol rather than the full wordmark - what do you, and the other @simple-icons/maintainers, think?

**Comment by wwwescape** [comment]
Sure. I'll take a look. I haven't had the time to look at the open PRs. Will do so in the coming weeks

**Comment by PeterShaggyNoble** [comment]
We do have precedent with airlines for just using the icon. See, for example, Air France & American Airlines.

**Comment by adamrusted** [comment]
I agree with @PeterShaggyNoble - we should just use the symbol as we have done with other airlines.

**Comment by wwwescape** [comment]
Will work on this in the next few weeks.

**Review by service-paradis — APPROVED** [review]
LGTM!

**Pull request merged**
